### PR TITLE
Factorize docker code

### PR DESCRIPTION
**Pull request by guhur**
## Problem

A few errors were reported when synchronizing the database when `django`

## Solution 

Following advises from 
[StackOverflow](https://stackoverflow.com/questions/35069027/docker-wait-for-postgresql-to-be-running), I used wait-for-it.sh to force synchronisation.

## Side-effect

It created again another file in the repository. I thought it would be time to refactor the docker files, since installation steps were anyway not so clear:
 
- the create-admin step is only required when accessing to the admin page
- the makemigration is not supposed to be run by anyone else than the project's devs
- the build step can be directly included inside the docker-compose

## Solution

I created a `docker` folder to move files inside and I modified the README.

BTW, I recommend to squash the commits when merging.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,25 +1,18 @@
-## Build the docker image
```
Comment (sebsheep) [inline README.md]: If I understand well, this build.sh script isn't useful anymore. In this case, it makes sense to remove this in the README.

However, we should not remove the other parts (migrations, adding a super user, ...) of the README except if these parts are obsolete, but in this case, we should add the new way of doing things.

File: .env [hunk 2]
```diff
@@ -2,3 +2,6 @@ SITE_URL=http://localhost:8000
 DEFAULT_FROM_EMAIL=dont-answer@mieuxvoter.fr
 EMAIL_HOST=mail
 EMAIL_PORT=25
+DJANGO_DEBUG=on
+DJANGO_ALLOWED_HOSTS=localhost
+DJANGO_SECRET_KEY=
```
Comment (sebsheep) [inline .env]: should belong to #14 ?

File: README.md (hunk 1 above)
Comment (guhur) [reply README.md]: Corrected in latest commit

File: requirements.txt [hunk 3]
```diff
@@ -1,7 +1,7 @@
-coverage==4.5.1
+coverage>=4.5.1
```
Comment (sebsheep) [inline requirements.txt]: I d prefere fixed versions, we wont have bad surprises if a lib upgrades and breaks its API

File: README.md [hunk 4]
```diff
@@ -1,25 +1,34 @@
...
 
-`sudo ./test.sh`
+```
+sudo docker exec -it mvapi_web_1 python ./manage.py createsuperuser
+```
 
-## Create a super-user
+Visit the admin page at [http://localhost:8012/admin/](http://localhost:8012/admin/).
 
-`sudo docker exec -it mvapi_web_1 python ./manage.py createsuperuser`
```
Comment (sebsheep) [inline README.md]: Why do you want to remove this?

Comment (guhur) [reply README.md]: I just moved it to distinct the installation steps, from the administration stuffs.

**Pull request merged**